### ISSUE TITLE
Incredible work!

### ISSUE THREAD

**Issue by loganpowell** [r-park/soundcloud-redux#15]
Your code and your UI is beautiful. Could you please share a high-level of your workflow? Do you start with mockups, then go to the reducers or do you start by designing the state then go to the actions then to sagas? Something completely different? How do you build out your first feature? Do you go from the bottom up or the top down?

**Comment by bodadotsh** [comment]
Agreed, also where's the homepage Featured Tracks coming from, e.g. like a playlist? Too bad I can't find it in the code...

**Comment by erubescent** [comment]
Featured Tracks are fetched from the `FEATURED_TRACKLIST_USER_ID` in [constants.js](https://github.com/r-park/soundcloud-redux/blob/master/src/core/constants.js). @bodaz

**Comment by henry-hz** [comment]
Amazing code! this is a real design. You are a great developer and artist.
I was wandering how to port your project to ReasonML.... any thoughts !!!
